components/ui/DataTable/ActionsDropdown.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuTrigger,
+} from '~/components/ui/dropdown-menu';
+import { MoreHorizontal } from 'lucide-react';
+import { Button } from '~/components/ui/Button';
+
+type MenuItem = 'Edit' | 'Delete' | 'Resume';
```

Comment:
It would be good if this could accept a generic so that the actions could be dynamically defined depending on the use-case.

Reply:
resolved with [fc91b0e](https://github.com/complexdatacollective/Fresco/pull/6/commits/fc91b0e8bb86177ae4d5927955474280dd03cc75)